database-mysqldb/Accounts.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+const db = require('./index.js');
+
+const Account = db.define('account', {
+  account_number: {
+    type: Sequelize.STRING(8),
+    primaryKey: true,
+    unique: true,
+    allowNull: false,
+  },
+  buying_power: {
+    type: Sequelize.DECIMAL(14, 4),
+  },
```

Comment:
@iamjaytong @wwong007 If the only setting we're defining is the 'type', we don't need to put it in an object. This will work the same:
```
buying_power: Sequelize.DECIMAL(14, 4),
```